spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -387,6 +387,12 @@ public class SendingBean {
 }
 ----
 
+===== Spring application context for Spring Cloud Stream applications
```

Comment:
Not sure if we need to add this as a documentation feature. Probably part of the upgrade guide.